components/atoms/TextInput/text-input.tsx

```diff
@@ -72,10 +72,10 @@ const TextInput = ({
           {!disabled && (
             <>
               {state === "valid" ? (
-                <CheckCircleFillIcon className="text-light-orange-9" size={14} />
+                <CheckCircleFillIcon className="text-light-orange-9" size={12} />
```

Comment:
Should this be configurable? How does this change impact where its being used?

Reply:
The icon looks too big in the inputs